service/build.gradle

```diff
@@ -318,6 +318,8 @@ dependencies {
 
   implementation group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.1'
 
+  implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-joda', version: '2.12.3'
```

Comment:
joda time is used by gov notify transfer object. new spring boot does not bring joda jaskson module by default anymore